src/main/java/com/rakovets/course/java/core/practice/date_and_time/TemporalAdjusterImplementationTwo.java

```diff
@@ -0,0 +1,16 @@
+package com.rakovets.course.java.core.practice.date_and_time;
+
+import java.time.temporal.ChronoUnit;
+import java.time.temporal.Temporal;
+import java.time.temporal.TemporalAdjuster;
+
+public class TemporalAdjusterImplementationTwo implements TemporalAdjuster {
+    int numberOfDays;
+    public TemporalAdjusterImplementationTwo(int numberOfDays){
```

Comment:
"Violation: There isn't separating white spaces for curly brake
Solution: Add white spaces"